tests/e2e/cucumber/features/smoke/urlJourneys.feature

```diff
@@ -19,21 +19,38 @@ Feature: web can be navigated through urls
     And "Alice" creates the following files into personal space using API
       | pathToFile | content     |
       | lorem.txt  | new content |
-    When "Alice" navigates to "versions" details panel of file "lorem.txt" through the URL
-    And "Alice" restores following resources
+    When "Alice" navigates to "versions" details panel of file "lorem.txt" of space "personal" through the URL
+    Then "Alice" restores following resources
       | resource  | to | version | openDetailsPanel |
       | lorem.txt | /  | 1       | false            |
-    When "Alice" navigates to "sharing" details panel of file "lorem.txt" through the URL
-    And "Alice" shares the following resource using the direct url navigation
+    When "Alice" navigates to "sharing" details panel of file "lorem.txt" of space "personal" through the URL
+    Then "Alice" shares the following resource using the direct url navigation
       | resource  | recipient | type | role     | resourceType |
       | lorem.txt | Brian     | user | Can view | file         |
-    When "Alice" opens the file "lorem.txt" through the URL
+    When "Alice" opens the file "lorem.txt" of space "personal" through the URL
     Then "Alice" is in a text-editor
     And "Alice" closes the file viewer
-    When "Alice" opens the folder "FOLDER" through the URL
+    When "Alice" opens the folder "FOLDER" of space "personal" through the URL
     And "Alice" opens the following file in texteditor
       | resource               |
       | file_inside_folder.txt |
     Then "Alice" is in a text-editor
     And "Alice" closes the file viewer
-    And "Alice" logs out
+    Then "Alice" logs out
+    When "Admin" assigns following roles to the users using API
+      | id    | role        |
+      | Alice | Space Admin |
+    And "Alice" creates the following project space using API
+      | name        | id     |
+      | Development | team.1 |
+    And "Alice" creates the following file in space "Development" using API
+      | name              | content                   |
+      | spaceTextfile.txt | This is test file. Cheers |
+    When "Alice" logs in
+    And "Alice" opens space "Development" through the URL
+    And "Alice" opens the following file in texteditor
+      | resource          |
+      | spaceTextfile.txt |
+    Then "Alice" is in a text-editor
+    And "Alice" closes the file viewer
+    Then "Alice" logs out
```

Comment:
I would create separate scenario here? or move please creating space to top of the test and delete logs out step

Reply:
I've combined them into a single one